docs/content/staking/faq.mdx

```diff
@@ -6,15 +6,23 @@ description: Frequently Asked Questions
 
 ### Where will users receive their staking reward for each staking option?
 
-If you staked using Flow Port, then you should be able to see your staking rewards there. You can also withdraw the rewards or manually re-stake them through Flow Port.
+Staking rewards are not deposited directly into a user's account.
+They are deposited to the user's rewards pool in their connected staking object
+and can be withdrawn or restaked at any time.
 
-If you staked using a staking provider such as Kraken, Blocto or Finoa, please read [our guides](/staking/custody-providers/) for more information.
+If you staked using Flow Port, then you should be able to see your staking rewards there.
+You can also withdraw the rewards or manually re-stake them through Flow Port.
+
+If you staked using a staking provider such as Kraken, Blocto or Finoa,
+please read [our guides](/staking/custody-providers/) for more information.
 
 ### Will staking rewards be automatically re-staked?
 
 There will be _no_ automatic re-staking of staking rewards with Flow Port (i.e. using Ledger or Blocto).
+If you want to re-stake your rewards, you must manually do so yourself.
 
-If you staked using a staking provider such as Kraken, Blocto or Finoa, please read [our guides](/staking/custody-providers/) for more information.
+If you staked using a staking provider such as Kraken, Blocto or Finoa,
+please read [our guides](/staking/custody-providers/) for more information.
```

Comment:
Do we want to make clear that DeFi liquid staking strategies are not in scope for node ops? https://app.increment.fi/staking

Reply:
added some clarification